frontend/packages/shared/src/queryInvalidator/SyncSuccessQueriesInvalidator.ts

```diff
@@ -24,6 +24,8 @@ export class SyncSuccessQueriesInvalidator extends Queue {
     'applicationmetadata.json': [QueryKey.AppMetadata, '[org]', '[app]'],
     'layout-sets.json': [QueryKey.LayoutSets, '[org]', '[app]'],
     'policy.xml': [QueryKey.AppPolicy, '[org]', '[app]'],
+    'Settings.json': [QueryKey.FormLayoutSettings, '[org]', '[app]', '[layoutSetName]'],
+    layouts: [QueryKey.FormLayouts, '[org]', '[app]', '[layoutSetName]'],
```

Comment:
Should we split this map into two parts? One for filenames and one for the folders, would that be more explicit about what it is? 😊 Right now it looks like the layouts is missing the file-extension, but it does not since it's a folder.

```typescript
 private readonly fileNameCacheKeyMap: Record<string, Array<QueryKey | string>> = {
    'applicationmetadata.json': [QueryKey.AppMetadata, '[org]', '[app]'],
    'layout-sets.json': [QueryKey.LayoutSets, '[org]', '[app]'],
    'policy.xml': [QueryKey.AppPolicy, '[org]', '[app]'],
    'Settings.json': [QueryKey.FormLayoutSettings, '[org]', '[app]', '[layoutSetName]'],
  };
  
   private readonly folderNameCacheKeyMap: Record<string, Array<QueryKey | string>> = {
     layouts: [QueryKey.FormLayouts, '[org]', '[app]', '[layoutSetName]'], 
  };
  ```

Reply:
Agree! But will it then also make sense changing the method that does this lookup? Maybe simply to `getCacheKeyByFileLocation` and the prop to `fileOrFolderName`? So we are consistent 😊